Create design history non-state-funded-schools-in-RECT.md (#373)
DFE-Digital/teacher-cpd-design-history: app/posts/ecf-v2/non-state-funded-schools-in-RECT.md

---
title: No longer asking non-state-funded schools to use the RECT service
description: "Why we’ve changed which types of schools can access the service."

> date: 2025-03-13

Reviewer: I wonder if we can just say 'unfunded' schools, or schools that are not funded for ECF.

---

> ## Why we're doing this

Author: ```suggestion
## Context
```

> ECTs can serve induction at non-state-funded schools, but they do not receive DfE funding for ECF training. These ECTs can still do unfunded training, using the publicly accessible materials.  

Reviewer: ```suggestion
ECTs can serve induction at schools that are not funded by the state, but they do not receive DfE funding for ECF training. These ECTs can still do unfunded training, either by using the publicly accessible materials from lead providers or by schools creating their own training following the initial teacher training early career framework.  
```

maybe?

> Mainly these schools are independent schools without section 41 status (c.1100 per year) and British schools overseas (c.900 per year).  

Reviewer: ```suggestion
Mainly these schools are independent schools without section 41 status (around 1100 per year) and British schools overseas (around 900 per year).  
```

> Previously, schools needed to use the service to gain access to the DfE materials if they chose this training option for their ECTs. However, these materials were made publicly available last year, so we wanted to explore whether there was still a need for non-state-funded schools (without section 41 funding) to use the RECT service at all. 

Reviewer: ```suggestion
Previously, schools needed to use the service to gain access to the DfE materials if they chose this training option for their ECTs. However, these materials were made publicly available last year, so we wanted to explore whether there was still a need for non-state-funded schools (without section 41 funding) to use the Register early career teachers service at all. 
```

Reviewer: Might also be worth explaining section 41?

Reviewer: or linking to an explanation

> 

Reviewer: Could we also add why this is beneficial? E.g. save money and time, less complexity in the journeys to register ECTs, only keep data we absolutely need in the service.

Here's the reasoning on our slides:

We are proposing that non state funded schools do not need to use the new Register ECTs service to report their ECTs and mentors. This is because:​

​

The data isn't needed to calculate funding as these ECTs and mentors do not receive funding​

We should try to reduce the burden for schools as much as possible, and only ask them to do work where it is purposeful​

For GDPR, we shouldn't collect and hold data (personal info such as name, email etc) in the service that we don't need​

We shouldn't spend a lot of money designing and building something different for the new service for non-state funded schools which doesn't ask for personal details and explains why we need the information​

ECTs at non state funded schools would still be registered by an AB for induction, so data on when the ECT started induction and what training programme type they're doing would still be available. We wouldn't know from the AB data which school these ECTs are at though.

Reviewer: just occurred to me... we're deduplicating registration for schools that aren't state-funded! (or section 41 funded!)

yippeeeee!

## What we did

We spoke to the DfE teams who use ECF schools data, so that we could understand if there was a need to collect non-state-funded schools (without section 41 funding) data specifically.  

We spoke to: 

> - Grant funding team: As these schools don't receive any funding, they didn't think they needed this data. 

Author: ```suggestion
**Grant funding team:** As these schools don't receive any funding, they didn't think they needed this data. 
```

> - TAD: The have a very small number of ECTs at technical colleges covered in their statistics, but they weren't sure why these ECTs were in scope. No other ECTs at non-state funded schools (except Section 41 schools) are included in their statistics. 

Author: ```suggestion
**TAD**: The have a very small number of ECTs at technical colleges covered in their statistics, but they weren't sure why these ECTs were in scope. No other ECTs at non-state funded schools (except Section 41 schools) are included in their statistics. 
```

> - ECF Policy Unit: They couldn’t see any major issues with not collecting this data. 

Author: ```suggestion
**ECF Policy Unit:** They couldn’t see any major issues with not collecting this data. 
```

> - Induction policy team: They also didn’t highlight any reasons for collecting this data. 

Author: ```suggestion
**Induction policy team:** They also didn’t highlight any reasons for collecting this data. 
```

> Following these conversations, we proposed that non-state-funded schools should **not** need to register their ECTs and mentors in RECT, for several reasons: 

Reviewer: oh this is here! I thought it would be in the why!

> **Grant funding team:** As these schools don't receive any funding, they didn't think they needed this data. 
> **TAD**: The have a very small number of ECTs at technical colleges covered in their statistics, but they weren't sure why these ECTs were in scope. No other ECTs at non-state funded schools (except Section 41 schools) are included in their statistics. 
> **ECF Policy Unit:** They couldn’t see any major issues with not collecting this data. 
> **Induction policy team:** They also didn’t highlight any reasons for collecting this data. 

Author: ```suggestion
**Grant funding team:** As these schools don't receive any funding, they didn't think they needed this data. 

**TAD**: The have a very small number of ECTs at technical colleges covered in their statistics, but they weren't sure why these ECTs were in scope. No other ECTs at non-state funded schools (except Section 41 schools) are included in their statistics. 

**ECF Policy Unit:** They couldn’t see any major issues with not collecting this data. 

**Induction policy team:** They also didn’t highlight any reasons for collecting this data. 
```
